cfbs/commands.py

```diff
@@ -1067,9 +1068,21 @@ def _compare_list(a, b):
             return 1
 
     path = os.path.join(name, "input.json")
-    log.debug("Writing json to file '%s'" % path)
-    write_json(path, data)
 
+    log.debug("Comparing with data already in file '%s'" % path)
+    old_data = read_json(path)
+    if old_data == data:
+        log.debug("Input data for '%s' unchanged, nothing to write / commit" % name)
+        return 0
+
+    log.debug("Input data for '%s' changed, writing json to file '%s'" % (name, path))
+    write_json(path, data)
+    if config.get("git", False):
+        git_commit(
+            "Set the input for module '%s'" % name,
+            edit_commit_msg=False,
```

Comment:
No interactivity here and prompting the user if they want to modify the commit message?

Reply:
Right, I am submitting some changes which should explain / clear this up.

Reply:
WIP: https://github.com/cfengine/cfbs/pull/145